.circleci/ci.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -ex
+cd "$(dirname "$0")"
+
+TENSORFLOW_BUILD_TAG="${TENSORFLOW_BUILD_TAG:-centos}"
+TENSORFLOW_DOCKER_BUILD_TAG="${TENSORFLOW_DOCKER_BUILD_TAG:-latest}"
+BAZEL_OUTPUT_BASE="${BAZEL_OUTPUT_BASE:-/var/tmp/bazel-output-base}"
+BUILD_CONFIG="${BUILD_CONFIG:-mkl}"
+
+if [[ ! -z "${BAZEL_VERSION}" ]]
+then
+    DOCKER_BUILD_ARGS="${DOCKER_BUILD_ARGS} --build-arg BAZEL_VERSION=${BAZEL_VERSION}"
+fi
+
+mkdir -p "${BAZEL_OUTPUT_BASE}"
+mkdir -p build
+
+
+
+mkdir -p "../${OUTPUT_DIR}"
```

Comment:
I'm a bit lost with the directory layout:
* build is created under .circleci
* OUTPUT_DIR (which is defined externally?) is created as sibling of .circleci and I'm not sure you need this